### PR TITLE
Create Python Quiz Game

### PR DESCRIPTION
**Pull request by Mikhaela-Paige**
Instructions on how to build a basic quiz game in python for beginners

### CODE REVIEW

File: Python Quiz Game [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+		        score += 1
+		        print(correct_answer)
+		    else:
+		        print(incorrect_answer)
+
+
+		print("⭐ Congratulations! You've made it! ⭐\nYou answered",score,"questions correctly!")
+		mark=(score/total_questions)*100
+		print('Total marks:',mark)
+		print('Thank you for playing, I hope you had fun!')
```
Comment (realstealthninja) [inline Python Quiz Game:57]: bad indentation

File: Python Quiz Game [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+		incorrect_answer = INCORRECT[num]
+
+
+		for question, right_answer in questions:
+		    answer = input(f"{question}")
+		    if answer.lower() == right_answer:
+		        score += 1
+		        print(correct_answer)
+		    else:
+		        print(incorrect_answer)
```
Comment (realstealthninja) [inline Python Quiz Game:51]:
```suggestion
		for question, right_answer in questions:
		    answer = input(f"{question}")
		    if answer.lower() == right_answer:
		        score += 1
		        print(correct_answer)
		        return
		    print(incorrect_answer)
```

File: Python Quiz Game [hunk 3]
```diff
@@ -0,0 +1,57 @@
+import random
+
+
+		print("Welcome to Q-Quiz!\nGood luck")
```
Comment (realstealthninja) [inline Python Quiz Game:4]: I'd prefer to wrap this under a function.

**Review by realstealthninja — CHANGES_REQUESTED** [review]

File: Python Quiz Game (hunk 1 above)
Comment (Mikhaela-Paige) [reply Python Quiz Game]: I wrote it locally and copied it over, that's why. Sorted now, thank you for pointing it out I didn't notice.

**Review by realstealthninja — COMMENTED** [review]
It's one space to the right now still going to error.

File: Python Quiz Game [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+  QUESTIONS = [
+      ('#question1', '#answer1'),
+      ('#question2', '#answer2'),
+      ('#question3', '#answer3'),
+  ]
+
+  num_questions=len(QUESTIONS)
+  questions = random.sample(list(QUESTIONS), k = num_questions)
+  num = random.randint(0,#n)
```
Comment (realstealthninja) [inline Python Quiz Game:21]: Why is #n here can't it be num_questions?
Or Len of the other list you created just above?

File: Python Quiz Game [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+  print("Welcome to Q-Quiz!\nGood luck")
+
+  answer=input("Ready?")
+  score=0
+
+  if answer.lower()=='yes':
+      print("Great!")
+  else:
+      print("See you later!")
```
Comment (realstealthninja) [inline Python Quiz Game:11]:
```suggestion
  if answer.lower() != 'yes':
      print("See you later!")
      return   #but convert the whole file into a function first
  print("Great!")
      
```
Use guard clauses where ever you can.

Comment (realstealthninja) [inline Python Quiz Game:11]: This else does nothing but print see you later and does not exit program

File: Python Quiz Game [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+  correct_answer = CORRECT[num]
+  incorrect_answer = INCORRECT[num]
+
+  for question, right_answer in questions:
+      answer = input(f"{question}")
+      if answer.lower() == right_answer:
+          score += 1
+          print(correct_answer)
+      else:
+          print(incorrect_answer)
```
Comment (realstealthninja) [inline Python Quiz Game:43]:
```suggestion
      if answer.lower() != right_answer:
          print(incorrect_answer)
          continue
      score += 1
      print(correct_answer)
          
```
Again guard clauses are always better

**Review by realstealthninja — CHANGES_REQUESTED** [review]
Some general comments on what the code is trying to do would be nice

File: Python Quiz Game (hunk 1 above)
Comment (Mikhaela-Paige) [reply Python Quiz Game]: suggestion outdated, conversation marked resolved

File: Python Quiz Game (hunk 2 above)
Comment (Mikhaela-Paige) [reply Python Quiz Game]: suggestion outdated, conversation marked resolved

File: Python Quiz Game (hunk 4 above)
Comment (Mikhaela-Paige) [reply Python Quiz Game:21]: I actually only started python a few weeks ago, and am not familiar with guard clauses. I shall research and then rectify.

Comment (Mikhaela-Paige) [reply Python Quiz Game]: While loop used for "Ready?" user input

**Comment by Mikhaela-Paige** [comment]
@cclauss Do you know of any other repositories that are partaking in Hacktoberfest in this case? This is the only one I have managed to find so far out of dozens I have tried that isn't marked as excluded.

**Comment by cclauss** [comment]
https://github.com/MakeContributions

**Pull request closed without merging**